apps/macos/package.json

```diff
@@ -21,38 +21,43 @@
   },
   "dependencies": {
     "@fluentui-react-native/tester": "^0.28.0",
-    "react-native-macos": "0.63.33",
-    "react-native-svg": "^12.1.1",
-    "react-native-test-app": "^0.3.2"
+    "react": "16.13.1",
+    "react-native-macos": "^0.63.0",
+    "react-native-svg": "^12.1.1"
   },
   "devDependencies": {
     "@babel/core": "^7.6.2",
     "@babel/runtime": "^7.6.2",
+    "@react-native-community/eslint-config": "^1.1.0",
+    "@rnx-kit/cli": "^0.5.26",
     "@uifabricshared/build-native": "^0.1.1",
     "@uifabricshared/eslint-config-rules": "^0.1.1",
-    "@react-native-community/eslint-config": "^1.1.0",
-    "@rnx-kit/cli": "^0.0.3",
     "babel-jest": "^24.9.0",
     "jest": "^24.9.0",
     "metro-config": "^0.58.0",
     "metro-react-native-babel-preset": "^0.59.0",
     "react-native": "^0.63.4",
     "react-native-svg-transformer": "^0.14.3",
+    "react-native-test-app": "^0.5.9",
     "react-test-renderer": "~16.13.1"
   },
-  "peerDependencies": {
-    "react": "^16.13.1"
-  },
   "jest": {
     "preset": "react-native"
   },
   "rnx-kit": {
+    "reactNativeVersion": "^0.63",
+    "kitType": "app",
     "bundle": {
       "targets": [
         "macos"
       ],
       "entryPath": "index.js"
-    }
+    },
+    "capabilities": [
+      "core-macos",
+      "react",
+      "test-app"
+    ]
```

Comment:
purposefully not bringing hermes to macOS and iOS?

Reply:
If we want to bring in Hermes, I would do it in a separate PR as it's more controversial.

Reply:
Just making sure =) 
We purposefully chose not to use Hermes on the apple platforms just yet. Also, Hermes on arm64 Macs won't work till RN 0.65...